website/docs/workloads/blenderbenchmark/blenderbenchmark.md

```diff
@@ -4,17 +4,16 @@ Blender Benchmark, a new platform to collect and display the results of hardware
 * [Blender Benchmark] (https://www.blender.org/news/introducing-blender-benchmark/)
 
 ## What is Being Measured 
-The Blender Benchmark Score is a measure of how quickly cycles can render [path tracing samples](https://docs.blender.org/manual/en/latest/render/cycles/render_settings/sampling.html) on one CPU or GPU device.
+The Blender Benchmark Score is a measure of how quickly cycles can render [path tracing samples](https://docs.blender.org/manual/en/latest/render/cycles/render_settings/sampling.html) using only the device specified. (E.g. CPU or GPU).
 
-The Blender Benchmark firstly downloads the blender engine and the scenes to be tested. The benchmark then tests how quickly these scenes can be rendered on one CPU or GPU device.
+**Note**: Integrated GPU may not be detectable by blender and thus may not be used for benchmarking. 
+
+The Blender Benchmark firstly downloads the blender engine and the scenes to be tested. The benchmark then tests how quickly these scenes can be rendered using, and only using, the specified device.
 
 The higher the number, the better. In particular it's the estimated number of samples per minute, summed for all benchmark scenes.
 
 ## System Requirements
 * Windows 8.1, 10, and 11
-* 64-bit quad core CPU with SSE2 support
```

Comment:
Although the Blender engine requires this system config, I was able to get blender benchmark results on a 2 core, 4 GM windows VM.